### ISSUE TITLE
Resource and scope validation

### ISSUE THREAD

**Issue by silvios** [bshaffer/oauth2-server-php#403]
I have been very impressed with how this is coded. Just incredible. So I have a working test api server, once I get a valid token and I make a resource request, I limit the source using scopes. Basically what I noticed is this the proper way to limit a resource by several scopes?

if ($server->verifyResourceRequest($app['request'], $response, "x") && $server->verifyResourceRequest($app['request'], $response, "x.readonly")) {
//return valid stuff
}

I believe it should work like this:

if ($server->verifyResourceRequest($app['request'], $response, "x x.readonly")) {
//return valid stuff
}

 Basically, I think all scopes should be separated by a space so you only have to run the check once, rather than re-running it again for all scopes that are allowed to view a resource. Also because of how the spec of oauth works when you request scopes in a url you do use a space to separate more than 1 scope. This should be a valid way of validating scopes.

If I missed something sorry. If you agree with the change and could point me in the right direction I do not mind making the required changes in order to make the first example work. 

Let me know what you think, thanks!
